api/rpc/stack/parse.go

```diff
@@ -68,7 +70,7 @@ func ParseStackfile(ctx context.Context, in string) (*Stack, error) {
 	var stack = &Stack{}
 	specs, err := parseStack([]byte(in))
 	if err != nil {
-		return nil, err
+		return nil, grpc.Errorf(codes.InvalidArgument, "%v", err)
```

Comment:
oh yes, excellent, i didn't see that it has been moved client side. Good! 
I updated back the pr for parse.go